api/symphony-api-no-k8s-munchen.json

```diff
@@ -66,10 +66,10 @@
               "baseUrl": "http://localhost:8082/v1alpha2/",
               "user": "admin",
               "password": "",
-              "providers.state": "k8s-state"        
+              "providers.volatilestate": "memory"        
```

Comment:
catalog manager expect persistent state provider.

Reply:
This is stage manager